config/routes.rb

```diff
@@ -271,6 +271,10 @@
     scope '/verify', as: 'idv' do
       get '/' => 'idv#index'
       get '/activated' => 'idv#activated'
+
+      if IdentityConfig.store.idv_api_enabled
+        get '/:step' => 'idv#show', constraints: { step: /password/ }
```

Comment:
As mentioned in https://github.com/18F/identity-idp/pull/6177/files#r845309695, the idea is we'd only want a single action to handle every step in the flow, since it's just initializing the React app. That's why there's a constraint here, even though there's (currently) only a single step being handled.

Reply:
Re-thinking this: Yes, technically, it can be shared, but the "constraint" patterns are probably going to be more cumbersome to maintain (and maybe less performant?) than to just list them out explicitly, and point to the same controller+action.